tasks/insights-client-unregistration.yml

```diff
@@ -11,3 +11,9 @@
       or "Registered" in __rhc_insights_status.stdout
   register: __rhc_insights_unregister_result
   changed_when: true
+
+- name: Reset ansible host in config
+  lineinfile:
```

Comment:
> Does Ansible have a plugin to handle INI files, so it’s not necessary to match the lines manually with a regexp? 

Yes and no - there is the community.general.ini_file module - but we really don't want the role to rely on 3rd party code unless absolutely necessary.

> Would this be sufficent with control characters that need to be escaped?

What is the issue?

Reply:
Oh. I didn’t kno that it’s 3rd party and we don’t want to rely on that.

For the escaping issue. I am not sure if there is any: I don’t know  by heard the details of how Python parses INI files. I was just afraid whether there aren’t some control characters that would cause that Python would read the value differently than hot it’s written without escaping.

Reply:
I suppose it is possible that someone could provide an `ansible_host` value with control characters, but that seems highly unlikely to me.

Reply:
I agree it’s unlikely. Maybe it’s not worth the effort. I would however expect from an official RHC role to be safe.